nomad/assets/dashboards/overview.json

```diff
@@ -0,0 +1,867 @@
+            "width": 20,
+            "x": 93,
+            "y": 0
+        },
+        {
+            "bgcolor": "white",
+            "font_size": "14",
+            "height": 5,
+            "html": "Broker Metrics",
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 18,
+            "x": 1,
+            "y": 52
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.nomad.broker.total_ready{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "# Brokers Ready",
+            "type": "query_value",
+            "width": 20,
+            "x": 0,
+            "y": 58
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.nomad.broker.total_blocked{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "# Brokers Blocked",
+            "type": "query_value",
+            "width": 20,
+            "x": 0,
+            "y": 71
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.nomad.broker.total_unacked{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "# Brokers Unacked",
+            "type": "query_value",
+            "width": 20,
+            "x": 0,
+            "y": 84
+        },
+        {
+            "bgcolor": "white",
+            "font_size": "14",
+            "height": 5,
+            "html": "RPC Metrics",
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 18,
+            "x": 22,
+            "y": 52
+        },
+        {
+            "bgcolor": "white",
```

Comment:
`Consul metrics` can be either gray or pink.

Pink is usually the color we use to reference addtl integration metrics but since these are emitted by nomad, they can also just be a category.
https://datadoghq.dev/integrations-core/guidelines/dashboards/#other-technologies